impl/src/main/java/org/jboss/weld/injection/InterceptionFactoryImpl.java

```diff
@@ -137,4 +169,25 @@ public T createInterceptedInstance(T instance) {
         return proxy;
     }
 
+    private boolean isInterceptorBinding(Annotation annotation) {
+        return annotation.annotationType().getAnnotation(InterceptorBinding.class) != null;
```

Comment:
You should probably use `BeanManager#isInterceptorBinding(Class<? extends Annotation>)` instead..

Reply:
Good point, will look at that.